libs/datamodel/core/tests/renderer/simplification.rs

```diff
@@ -19,12 +19,12 @@ fn test_must_not_render_relation_fields_with_many_to_many() {
     let expected = expect![[r#"
         model Post {
           id   Int    @id @default(autoincrement())
-          User User[]
+          User User[] @relation(references: [id])
         }
 
         model User {
           id   Int    @id @default(autoincrement())
-          Post Post[]
+          Post Post[] @relation(references: [id])
```

Comment:
The responsibility of not populating references now falls on the introspection engine (it's trivial there). The references arg is there in this test because lift adds it.